app/views/orders/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Your Order</h1>
```

Comment:
THe show page has unique requirements per the specs, so would like to opt out of the partial in this case.